rest/server_context.go

```diff
@@ -685,10 +682,39 @@ func dbcOptionsFromConfig(sc *ServerContext, config *DbConfig, dbName string) (d
 		compactIntervalSecs = uint32(*config.CompactIntervalDays * 60 * 60 * 24)
 	}
 
-	resyncQueryLimit := db.DefaultResyncQueryLimit
-	if config.ResyncQueryLimit != nil {
-		resyncQueryLimit = *config.ResyncQueryLimit
+	var queryPaginationLimit *int
```

Comment:
Is there a reason this needs to be *int instead of int?  It seems like it's just adding complexity, and the eventual value we're going to use is just int.

Reply:
Switched to int and using 0 to check for unset rather than nil. This means if 0 is set it'll use the default though which is probably fine